mreg_cli/cli.py

```diff
@@ -218,6 +218,20 @@ def _quit(args):
     callback=_quit,
 )
 
+cli.add_command(
+    prog='quit',
+    description='Exit application.',
+    short_desc='Quit',
+    callback=_quit,
+)
+
```

Comment:
Duplicate.

Reply:
Yeah, the request was to have a command `exit` work exactly as `quit`, so I reused the callback.

Reply:
Bah. Mea culpa. Fixed in https://github.com/unioslo/mreg-cli/pull/163/commits/08c0ae5d06c1122f1eaeb23116b27438caf7dade